### ISSUE TITLE
Signature Capture

### ISSUE THREAD

**Issue by crazyk357** [bizswoop/Point-of-Sale-POS-WooCommerce#14]
The ability to use both touch screen and other hardware like topaz signature pads to take signatures from customers that will show on the receipt?
Please let me know what you think. I would love to help!
Keven

**Comment by ncovercash** [comment]
Already ahead of you - #11  (although no more progress has been made there, lol)

**Comment by SupportBizSwoop** [comment]
Correct, on the integration of any touch screen or signature pads, we unfortunately haven't made any progress. The challenge integrations of hardware and SDKs take a significant amount of development work and unless we have a large request for this by the user base on the same device, it's hard for us to squeeze into our roadmap and allocate resources. 

Understanding we can be a bottleneck for hardware integrations, we are working on a developers kit for the POS. Allowing community members and developers to start working to develop around the POS with Hardware. We are hoping to support the developers kit next year, after we deploy our new Cloud and Server based configuration options for the POS.
